src/include/units/quantity_cast.h

```diff
@@ -341,8 +341,8 @@ template<Kind ToK, Unit ToU, typename K, typename U, typename Rep>
  *
  * @tparam CastSpec a target (quantity) point kind type to cast to or anything that works for quantity_kind_cast
  */
-template<typename CastSpec, typename PK, typename U, typename Rep>
-[[nodiscard]] constexpr QuantityPointKind auto quantity_point_kind_cast(const quantity_point_kind<PK, U, Rep>& qpk)
+template<typename CastSpec, typename PK, typename U, typename Rep, typename Org>
```

Comment:
Please decide if you prefer to use `O` or `Org` for origin. For me personally `Org` looks more like `original`.

Reply:
Organization for me.

Reply:
Which one would you prefer? I also had toyed with `Origin`, but I feared that it is too long for a likely appearance in many template instantiation traces. On the other hand `O` is too similar to `0` and maybe a bit too short for something not quite as central as `D` and `U`. Maybe `Orig`?  Or rename the concept completely to something like `PointReference` and then go with `Ref` (probably too close to `Rep`).

Reply:
`Orig`'s good enough for me. Then `O` could still be reserved for something more encompassing than just quantity points, like `D` and `U` are.

Reply:
`Orig` seems OK